.pre-commit-config.yaml

```diff
@@ -0,0 +1,29 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 22.8.0
+    hooks:
+      - id: black
+        # It is recommended to specify the latest version of Python
+        # supported by your project here, or alternatively use
+        # pre-commit's default_language_version, see
+        # https://pre-commit.com/#top_level-default_language_version
+        files: 'pydanticrud/.*\.py$'
+        language_version: python3.8
```

Comment:
Just want to make sure we don't want to specify 3.10 here since it's the latest version in the test matrix and suggested by the comment.

Reply:
This gets a bit dicey with the lowest version being 3.7. I don't want to rise the lowest version too quickly just yet.